eo-runtime/src/test/java/org/eolang/PhiTest.java

```diff
@@ -42,12 +42,15 @@ public void takesPackage() {
                     new PhMethod(
                         new PhMethod(
                             new PhMethod(
-                                Phi.Φ.attr("org").get(),
-                                "eolang"
+                                new PhMethod(
+                                    Phi.Φ.attr("org").get(),
+                                    "eolang"
+                                ),
+                            "math"
                             ),
-                        "math"
+                            "random"
```

Comment:
@mximp why 8 indents?

Reply:
@yegor256 didn't get that - parameters are indented one under another.
Please clarify.